test/rules/prefer_if_elements_to_map_literals.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore_for_file: prefer_expression_function_bodies
+// ignore_for_file: avoid_positional_boolean_parameters
```

Comment:
Do we need this ignore? I don't see any boolean parameters.

Reply:
Nope.  You're right.  Another copy-paste goof-up.